From 2b23effd5e5e1309501eeab4d07d01cf20a366db Mon Sep 17 00:00:00 2001
From: Min RK <benjaminrk@gmail.com>
Date: Tue, 25 Jun 2024 11:29:03 +0000
Subject: [PATCH] accept stable dijitso 2019.1.0

there are no commits in 2019.2.dev, so no reason to reject the stable release, which is identical
---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index c481f341..5caca8e4 100755
--- a/setup.py
+++ b/setup.py
@@ -14,6 +14,7 @@ on_rtd = os.environ.get('READTHEDOCS') == 'True'
 
 VERSION = "2019.2.0.dev0"
 RESTRICT_REQUIREMENTS = ">=2019.2.0.dev0,<2019.3"
+DIJITSO_RESTRICT_REQUIREMENTS = ">=2019.1.0,<2019.3"
 UFL_RESTRICT_REQUIREMENTS = ">=2022.3.0" # version of new ufl_legacy
 
 if on_rtd:
@@ -23,7 +24,7 @@ else:
         "numpy",
         "fenics-fiat%s" % RESTRICT_REQUIREMENTS,
         "fenics-ufl-legacy%s" % UFL_RESTRICT_REQUIREMENTS,
-        "fenics-dijitso%s" % RESTRICT_REQUIREMENTS,
+        "fenics-dijitso%s" % DIJITSO_RESTRICT_REQUIREMENTS,
     ]
 
 
-- 
2.45.0

